modules/nw-ptp-operator-metrics-reference.adoc

```diff
@@ -23,9 +18,38 @@ Some of the following metrics are applicable for PTP grandmaster clocks (T-GM) o
 |`openshift_ptp_clock_class`
 |Returns the PTP clock class for the interface.
 Possible values for PTP clock class are 6 (`LOCKED`), 7 (`PRC UNLOCKED IN-SPEC`), 52 (`PRC UNLOCKED OUT-OF-SPEC`), 187 (`PRC UNLOCKED OUT-OF-SPEC`), 135 (`T-BC HOLDOVER IN-SPEC`), 165 (`T-BC HOLDOVER OUT-OF-SPEC`), 248 (`DEFAULT`), or 255 (`SLAVE ONLY CLOCK`).
-Applicable to T-GM clocks only.
 |`{node="compute-1.example.com",process="ptp4l"} 6`
 
+|`openshift_ptp_process_restart_count`
+|Returns a count of the number of times the `ptp4l` process was restarted.
+|`{config="ptp4l.0.config", node="compute-1.example.com",process="phc2sys"} 1`
+
+|`openshift_ptp_threshold`
+a|Returns values for `HoldOverTimeout`, `MaxOffsetThreshold`, and `MinOffsetThreshold`.
+
+* `holdOverTimeout` is the time value in seconds before the PTP clock event state changes to `FREERUN` when the PTP master clock is disconnected.
+* `maxOffsetThreshold` and `minOffsetThreshold` are offset values in nanoseconds that compare against the values for `CLOCK_REALTIME` (`phc2sys`) or master offset (`ptp4l`) values that you configure in the `PtpConfig` CR for the NIC.
+|`{node="compute-1.example.com", profile="grandmaster", threshold="HoldOverTimeout"} 5`
+
+|`openshift_ptp_ha_profile_status`
+|Returns the current status of the highly available system clock when there are multiple time sources on different NICs.
+Possible values are 0 (`INACTIVE`), and 1 (`ACTIVE`).
+|`{node="node1",process="phc2sys",profile="profile1"} 1`
+`{node="node1",process="phc2sys",profile="profile2"} 0`
+|====
+
+[discrete]
+== PTP fast event metrics when T-GM is enabled
+
+The following table describes the PTP fast event metrics that are available only when PTP grandmaster clock (T-GM) is enabled.
+
+.PTP fast event metrics when T-GM is enabled
+[cols="1,4,3", options="header"]
+|====
+|Metric
+|Description
+|Example
+
 |`openshift_ptp_clock_state`
 |Returns the current PTP clock state for the interface.
```

Comment:
openshift_ptp_clock_state is available for all configuration including T-GM its not just for GM .
This metrics  identifies synchronization state of  PHC , OS Clock  (CLOCK_REALTIME), 
Not for T-GM: ts2phc lock state, DPLL Phase lock state state, DPLL Frequency Lock State, GPS FIX state etc

Reply:
Updated as per the suggestion.